docker/.env

```diff
@@ -69,75 +82,114 @@ MF_EMAIL_PASSWORD=2b0d302e775b1e
 MF_EMAIL_FROM_ADDRESS=from@example.com
 MF_EMAIL_FROM_NAME=Example
 MF_EMAIL_TEMPLATE=smtp-notifier.tmpl
-MF_EMAIL_INSTANCE_ID=
-
-### Token utility
-MF_TOKEN_RESET_ENDPOINT=/reset-request
 
 ### Things
 MF_THINGS_LOG_LEVEL=debug
+MF_THINGS_STANDALONE_ID=
+MF_THINGS_STANDALONE_TOKEN=
+MF_THINGS_CACHE_KEY_DURATION=10m
+MF_THINGS_HTTP_HOST=things
 MF_THINGS_HTTP_PORT=9000
-MF_THINGS_AUTH_HTTP_PORT=9001
-MF_THINGS_AUTH_GRPC_PORT=7000
-MF_THINGS_AUTH_GRPC_URL=things:7000
-MF_THINGS_AUTH_GRPC_TIMEOUT=1s
+MF_THINGS_HTTP_SERVER_CERT=
+MF_THINGS_HTTP_SERVER_KEY=
+MF_THINGS_GRPC_HOST=things
+MF_THINGS_GRPC_PORT=7000
+MF_THINGS_GRPC_SERVER_CERT=
+MF_THINGS_GRPC_SERVER_KEY=
+MF_THINGS_ES_URL=es-redis:${MF_REDIS_TCP_PORT}
+MF_THINGS_ES_PASS=
+MF_THINGS_ES_DB=0
+MF_THINGS_CACHE_URL=things-redis:${MF_REDIS_TCP_PORT}
+MF_THINGS_CACHE_PASS=
+MF_THINGS_CACHE_DB=0
 MF_THINGS_DB_HOST=things-db
 MF_THINGS_DB_PORT=5432
 MF_THINGS_DB_USER=mainflux
 MF_THINGS_DB_PASS=mainflux
 MF_THINGS_DB=things
-MF_THINGS_URL=http://mainflux-things:9000
-MF_THINGS_STANDALONE_ID=
-MF_THINGS_STANDALONE_TOKEN=
-MF_THINGS_CACHE_KEY_DURATION=10m
-MF_THINGS_CACHE_URL=things-redis:${MF_REDIS_TCP_PORT}
-MF_THINGS_CACHE_PASS=
-MF_THINGS_CACHE_DB=
-MF_THINGS_ES_URL=es-redis:${MF_REDIS_TCP_PORT}
-MF_THINGS_ES_PASS=
-MF_THINGS_ES_DB=
+MF_THINGS_DB_SSL_MODE=disable
+MF_THINGS_DB_SSL_CERT=
+MF_THINGS_DB_SSL_KEY=
+MF_THINGS_DB_SSL_ROOT_CERT=
 MF_THINGS_INSTANCE_ID=
 
+#### Things Client Config
+MF_THINGS_URL=http://things:9000
+MF_THINGS_AUTH_GRPC_URL=things:7000
+MF_THINGS_AUTH_GRPC_TIMEOUT=1s
+MF_THINGS_AUTH_GRPC_CLIENT_TLS=
+MF_THINGS_AUTH_GRPC_CA_CERTS=
+
 ### HTTP
+MF_HTTP_ADAPTER_LOG_LEVEL=debug
+MF_HTTP_ADAPTER_HOST=http-adapter
 MF_HTTP_ADAPTER_PORT=8008
+MF_HTTP_ADAPTER_SERVER_CERT=
+MF_HTTP_ADAPTER_SERVER_KEY=
 MF_HTTP_ADAPTER_INSTANCE_ID=
 
 ### MQTT
 MF_MQTT_ADAPTER_LOG_LEVEL=debug
 MF_MQTT_ADAPTER_MQTT_PORT=1883
-MF_MQTT_BROKER_PORT=1883
+MF_MQTT_ADAPTER_MQTT_TARGET_HOST=vernemq
+MF_MQTT_ADAPTER_MQTT_TARGET_PORT=1883
+MF_MQTT_ADAPTER_FORWARDER_TIMEOUT=30s
+MF_MQTT_ADAPTER_MQTT_TARGET_HEALTH_CHECK=http://vernemq:8888/health
 MF_MQTT_ADAPTER_WS_PORT=8080
-MF_MQTT_BROKER_WS_PORT=8080
-MF_MQTT_ADAPTER_ES_DB=0
+MF_MQTT_ADAPTER_WS_TARGET_HOST=vernemq
+MF_MQTT_ADAPTER_WS_TARGET_PORT=8080
+MF_MQTT_ADAPTER_WS_TARGET_PATH=/mqtt
+MF_MQTT_ADAPTER_INSTANCE=
+MF_MQTT_ADAPTER_ES_URL=es-redis:${MF_REDIS_TCP_PORT}
 MF_MQTT_ADAPTER_ES_PASS=
 MF_MQTT_ADAPTER_INSTANCE_ID=
+MF_MQTT_ADAPTER_ES_DB=0
 
 ### VERNEMQ
 MF_DOCKER_VERNEMQ_ALLOW_ANONYMOUS=on
 MF_DOCKER_VERNEMQ_LOG__CONSOLE__LEVEL=error
 
 ### CoAP
 MF_COAP_ADAPTER_LOG_LEVEL=debug
+MF_COAP_ADAPTER_HOST=coap-adapter
 MF_COAP_ADAPTER_PORT=5683
+MF_COAP_ADAPTER_SERVER_CERT=
+MF_COAP_ADAPTER_SERVER_KEY=
```

Comment:
@rodneyosodo , This two varaibles are needed 
because you have added `MF_COAP_ADAPTER_HTTP_SERVER_CERT`

Reply:
Yes. They are added already

Reply:
@rodneyosodo , These two variables not used, Please double check again and remove them

Reply:
I have implemented it to use `TLS` if they are provided

Reply:
Okay, Got it